src/test/java/PuppyTest.java

```diff
@@ -0,0 +1,19 @@
+
+import org.example.pages.PuppySearchPage;
+import org.junit.jupiter.api.Test;
+
+
+public class PuppyTest extends BaseTest {
+    private final PuppySearchPage puppySearchPage = new PuppySearchPage();
+    private static final String USERNAME = "admin";
+    private static final String PASSWORD = "password";
+
+    @Test
+    public void test3() {
```

Comment:
We can add some meaningful name like loginTest for example

Reply:
Why loginTest, we will test something else?

Reply:
If we will do something else test anywhere will named by loginTest? You said it just start for all of us.